### PR TITLE
Add testcafe-testing-library

### PR DESCRIPTION
**Pull request by benmonro**
Add docs for testcafe-testing-library

### CODE REVIEW

File: docs/testcafe-testing-library/intro.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  addTestcafeTestingLibrary,
+} from 'testcafe-testing-library'
+
+fixture`selectors`.beforeEach(addTestcafeTestingLibrary)
+  .page`http://localhost:13370`
+```
+
+You can now import & use `getBy`, `getAllBy`, `queryBy` and `queryAllBy`
+selectors in your tests.
+[See `dom-testing-library` API for reference](dom-testing-library/api-queries.md)
```
Comment (kentcdodds) [inline docs/testcafe-testing-library/intro.md]: Should probably mention that these queries are all pre-bound to `document.body` so people don't assume they have to pass the `container` argument.

**Comment by benmonro** [comment]
@kentcdodds. Accidentally hit resolve on my phone but thats a great point.  Will add a note about that.

**Comment by benmonro** [comment]
@kentcdodds ready to merge.

**Comment by alexkrolick** [comment]
Cool!

**Pull request merged**